ads/google/a4a/traffic-experiments.js

```diff
@@ -184,21 +186,27 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
  *   parameter or not.
  */
 function maybeSetExperimentFromUrl(win, element, experimentName,
-    controlBranchId, treatmentBranchId, delayedControlId,
-    delayedTreatmentBrandId, sfgControlId, sfgTreatmentId, manualId) {
+     controlBranchId, treatmentBranchId, delayedControlId,
```

Comment:
Lint didn't complain... figured it was part of the new linter changes rolled out by Erwin.